src/Orc.Controls/Controls/Callout/Controls/ApplicationPopup.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace Orc.Controls
+{
+    using System;
+    using System.Windows;
+    using System.Windows.Controls.Primitives;
+    using System.Windows.Interop;
+
+    internal class ApplicationPopup : Popup
```

Comment:
Does this code come from somewhere? If so, mention the original source so we provide the correct credit to the right people.

Reply:
Compilation of different codes. Neither of ready to use code worked for me well.